src/components/TestDrive/TestDriveDemo.tsx

```diff
@@ -28,9 +28,9 @@ function TestDriveDemo() {
     const [introSelected] = useOnyx(ONYXKEYS.NVP_INTRO_SELECTED, {canBeMissing: true});
     const {testDrive} = useOnboardingMessages();
     const currentUserPersonalDetails = useCurrentUserPersonalDetails();
-    const [isPolicyAdmin = false] = useOnyx(ONYXKEYS.COLLECTION.POLICY, {
+    const [isUserPolicyAdmin = false] = useOnyx(ONYXKEYS.COLLECTION.POLICY, {
```

Comment:
```suggestion
    const [isCurrentUserPolicyAdmin = false] = useOnyx(ONYXKEYS.COLLECTION.POLICY, {
```

I think this is more unambiguous given that we have a function named `isUserPolicyAdmin` to check if _any_ user (not just the current user) is an admin.

Reply:
@daledah 

bump here.